content/docs/en/getting-started/2-playground-tutorial.md

```diff
@@ -229,6 +229,7 @@ new Vue({
       console.log('Task with index: ' + args.index + ' tapped'); // Logs tapped tasks in the console for debugging.
     },
     onButtonTap() {
+      if (this.textFieldValue === "") return // Prevent user input empty string
```

Comment:
Two things here:

* The semi-colon is missing at the end of the line
* Please, pay attention to the format of code comments above and below and fix it for consistency. Ideally, here you would go with something like "Prevents users from entering an empty string."

Reply:
Hi @ikoevska ,

Sorry for the mistakes, i've updated the code. Thanks